chaincode-labs/chaincode-podcast/chaincode-decoded-mempool.md

```diff
@@ -1,436 +1,417 @@
 
-Speaker 1: 00:00:31
+Mark Erhardt: 00:00:31
 
-We often talk about the mempool, but there is no such thing as a global mempool.
-Every full node has its own mempool and the mempool is basically just the queue of transactions waiting to get confirmed.
-Where confirmed means included in a block.
-So by default block template builders will just sort the waiting transactions by the highest effective fee rate, then pick from the top.
+We often talk about the mempool, but there is no such thing as a global mempool, every full node has its own mempool.
+The mempool is just the queue of transactions waiting to get confirmed, where confirmed means included in a block.
+By default, block template builders will sort the waiting transactions by the highest effective fee rate, then pick from the top.
 The juicier a transaction the quicker it gets confirmed.
-Now especially in the last few months we've seen that there was very large queues because we had a huge run up in the price I haven't checked, but I think it's now about 120 days that we haven't cleared the mempool, Maybe 110 since 15th of December.
-So mempools are limited and by default they are limited to 300 megabytes of deserialized data.
-So that includes all the overhead structure, the previous UTXOs, maybe even the whole transaction that created UTXOs and so forth.
-So roughly at about 80 blocks worth of data, the default of 300 megabyte gets exceeded, and at that point, a full node will automatically start purging the lowest fee rate transactions.
-They just drop them and tell all their neighboring peers, hey, don't send me anything under this fee rate.
+Now especially in the last few months we've seen that there was very large queues because we had a huge run up in the price - I haven't checked, but I think it's now about 120 days that we haven't cleared the mempool, maybe 110, since 15th of December.
+Mempools are limited, by default they are limited to 300 megabytes of deserialized data.
+That includes all the overhead structure, the previous UTXOs, maybe even the whole transaction that created UTXOs and so forth.
+So roughly at about 80 blocks worth of data, the default of 300 megabytes gets exceeded, and at that point, a full node will automatically start purging the lowest fee rate transactions.
+They just drop them and tell all their neighboring peers - "hey, don't send me anything under this fee rate."
 They start raising up their min fee rate.
-So the problem that gets introduced here is if a parent transaction is no longer in the mempool, you cannot bump it.
 
 ## Child Pays for Parent (CPFP)
 
-Speaker 1: 00:02:12
+Mark Erhardt: 00:02:04
+
+The problem that gets introduced here is if a parent transaction is no longer in the mempool, you cannot bump it - because if you try to do a CPFP and the parent isn't there, the child is going to be invalid.
+
+Host0:02:19
+
+[CPFP](https://bitcoinops.org/en/topics/cpfp/) just for the uninitiated, (is) child pays for parent.
```

Comment:
Just tested this out. Great addition, such links and collections are what make this format all the more useful.

Since the btctranscripts tags provide the Optech summary and link, as well as related transcripts, I see no reason not to prefer this approach. Replacing all Optech links which have a corresponding btctranscripts tag link could, as you mentioned, be easily automated whenever it is decided.